k8s/namespaces/admin/flux-helm-operator/rbac/prod-role-binding.yaml

```diff
@@ -557,3 +557,19 @@ subjects:
     kind: ServiceAccount
 
 ---
+
```

Comment:
This is for flux v1 and shouldn't be needed for a new app.

Reply:
@adusumillipraveen - Are you saying we don't need to add any RBAC role binding for Flux V2? In other words abandon this PR? Thanks

Reply:
@adusumillipraveen - Hi, did you see my reply to your comment?

Reply:
yes pls, this is only for v1.